packages/picasso-lab/src/Section/Section.tsx

```diff
@@ -136,8 +137,12 @@ export const Section = forwardRef<HTMLDivElement, Props>(function Section(
       {hasHeader && (
         <Container
           data-testid={testIds?.header}
-          className={cx(classes.header, {
-            [classes.collapsedHeader]: collapsed
+          className={cx({
+            [classes.header]: variant !== 'withHeaderBar',
+            [classes.headerBar]: variant === 'withHeaderBar',
+            [classes.collapsedHeader]: variant !== 'withHeaderBar' && collapsed,
+            [classes.collapsedHeaderBar]:
+              variant === 'withHeaderBar' && collapsed
```

Comment:
```suggestion
            [classes[`header${variant}`]]: true,
            [classes[`collapsedHeader${variant}`]]: collapsed,
```
wdyt, won't it be so confusing then?